src/shaders/symbol_icon.vertex.glsl

```diff
@@ -3,11 +3,20 @@ attribute vec4 a_pos_offset;
 attribute vec2 a_texture_pos;
 attribute vec4 a_data;
 
+// icon-size data (see symbol_sdf.vertex.glsl for more)
+attribute vec3 a_size;
```

Comment:
Yep we still need to combine https://github.com/mapbox/mapbox-gl-js/pull/4455#discussion_r108415748